rpc/src/client/channel.rs

```diff
@@ -250,7 +250,7 @@ where
     let (cancellation, canceled_requests) = cancellations();
     let canceled_requests = canceled_requests.fuse();
 
-    crate::spawn(
+    runtime::spawn(
```

Comment:
I'd like to keep `crate::spawn`, here and elsewhere. `runtime::spawn` panics on error and also sets up a channel for the response, neither of which we want.